modules/apps/portal-workflow/portal-workflow-kaleo-test/src/testIntegration/resources/com/liferay/portal/workflow/kaleo/dependencies/message-boards-moderation-workflow-definition.xml

```diff
@@ -0,0 +1,223 @@
+<?xml version="1.0"?>
```

Comment:
@mxs2 I believe we do not need this file because the workflow definition is already created on company creation.